pom.xml

```diff
@@ -93,6 +93,13 @@
           <includeTestResources>false</includeTestResources>
           <linkXRef>false</linkXRef>
         </configuration>
+         <dependencies>
+            <dependency>
+              <groupId>com.puppycrawl.tools</groupId>
+              <artifactId>checkstyle</artifactId>
+              <version>10.15.0</version>
```

Comment:
maybe extract the version to a property?

Reply:
I had considered it, but given that it's single use I didn't really see the advantage